godtools/App/Share/Domain/UseCases/GetShareableImageUseCase/ShareableImageModel.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  ShareableImageModel.swift
+//  godtools
+//
+//  Created by Rachael Skeath on 7/25/22.
+//  Copyright © 2022 Cru. All rights reserved.
+//
+
+import UIKit
+
+struct ShareableImageModel {
```

Comment:
Hey @rachaelblue can you change this name to ```ShareableImageDomainModel```.  That is something I would like to shift towards for our domain and data models.  Suffixing with either ```DomainModel``` or ```DataModel```.  That will help us see which is coming from the domain vs data layer.

Reply:
Ooo good idea, I was thinking about that too.  That'll help make things clearer for sure.